packages/spindle-mcp-server/src/index.ts

```diff
@@ -114,6 +115,21 @@ server.tool(
   }),
 );
 
+// アクセシビリティ関連のドキュメントを取得します
+server.tool(
+  'get_accessibility_docs',
+  'アクセシビリティ関連のドキュメントを取得します',
```

Comment:
Ameba独自のドキュメントなので
```suggestion
  'Spindle UI Componentで推奨するアクセシビリティのガイドラインを取得します',
```
とかのがよりよさそうです 😄

Reply:
一応、Ameba, Spindleでのというprefixはついた状態(他のコンポーネント、デザイントークン等も)ですが、どうですかね〜。というのとチェックリストだけでなく、a11y関連のドキュメントを返す tool の予定ではあります！

なお一気に返しちゃうのは方針で、toolを分けちゃうと明示的に指示しないと見に行かないことがあるからです。データサイズと引き換えに今は利便性をとっています〜 (将来変わるかもしれません)

https://github.com/openameba/spindle/blob/208a47dbb44ea6de7c9ee94c7c220e775a821626/packages/spindle-mcp-server/src/index.ts#L11-L13

https://github.com/openameba/spindle/pull/1311/files#diff-bfed0da5572cb3af753c29fa34e4a1dfdd26bc2bf3b36ef8066f297ff06f16e9R51

(他の項目も返せるようなobject)

Reply:
なるほど理解です！ 🙆‍♀️ 
WCAGとかWAI-ARIAとか必要に応じて足すかもってことですね 👍 👍